internal/server/get_payments_test.go

```diff
@@ -48,6 +48,34 @@ func TestGetPayments(t *testing.T) {
 			ID:     3,
 			Amount: 1438,
 		},
+		{
+			ID:               3,
```

Comment:
```suggestion
			ID:               4,
```
Just so theres no duplicate IDs in the test